examples/2d-meta_train.py

```diff
@@ -1,3 +1,6 @@
+import sys
+sys.path.append("/home/jieyi/rl4co")
```

Comment:
[Minor] remember to remove this - if you install `rl4co` locally in editable mode with `pip install -e .`, importing a path should not be needed

Reply:
Yes, already removed in the latest commit:)